### DISCUSSION TITLE
2023/01/03/%E5%A6%82%E4%BD%95%E9%80%9F%E8%AF%BB/

### DISCUSSION THREAD

**Discussion by giscus** [WyattJia/WyattJia#82] [Announcements]
# 2023/01/03/%E5%A6%82%E4%BD%95%E9%80%9F%E8%AF%BB/

https://eventloop.live/2023/01/03/%E5%A6%82%E4%BD%95%E9%80%9F%E8%AF%BB/

**Comment by cxybreader** [comment]
心得不错，但快速阅读不意味着快速记录，更不代表快速理解，共同进步
